spec/fs-plus-spec.coffee

```diff
@@ -639,3 +639,52 @@ describe "fs", ->
 
       expect(fs.existsSync(filePath)).toBe false
       expect(fs.existsSync(newFilePath)).toBe true
+
```

Comment:
GitHub's diffing didn't close the comment, but I removed the extra line you pointed out.